olive/model/utils/hf_utils.py

```diff
@@ -41,22 +37,32 @@ def load_hf_model_from_task(task: str, name: str, **kwargs):
     class_tuple = ()
     class_tuple = class_tuple + model_class.get("pt", (AutoModel,))
 
-    model = None
+    exceptions = []
     for model_class in class_tuple:
         try:
             model = model_class.from_pretrained(name, **kwargs)
             logger.debug("Loaded model %s with name_or_path %s", model_class, name)
             return model
-        except (OSError, ValueError):
+        except (OSError, ValueError) as e:
+            if len(class_tuple) == 1:
```

Comment:
what about we raise the exception if it is the last item in the tuple?
```python
for i, model_class in enumerate(class_tuple):
      except:
        if i == len(class_tuple) - 1:
           raise
```

Reply:
done